api/nodes/views.py

```diff
@@ -27,7 +30,6 @@ def get_node(self):
         self.check_object_permissions(self.request, obj)
         return obj
 
-
```

Comment:
FWIW, this rule is explicitly ignored by our setup.cfg: https://github.com/CenterForOpenScience/osf.io/blob/05d7f0e135a6bd5bb8ca196646bfbb85acf2dc25/setup.cfg#L6.

I find the extra whitespace to be unnecessary; it does not significantly affect readability.